package.json

```diff
@@ -5,7 +5,7 @@
   ],
   "scripts": {
     "dev": "yarn workspace @okta/vuepress-site dev",
-    "build": "yarn lint-all && yarn workspace @okta/vuepress-site build",
+    "build": "yarn workspace @okta/vuepress-site build",
```

Comment:
separation of these, so that a build doesn't ALWAYS fire a lint.  if this is too impacting to other development processes, happy to put back.

Reply:
Yes that makes sense. If we remove lint from build command we need to run lint check before [yarn build](https://github.com/okta/okta-developer-docs/blob/master/scripts/publish.sh#L24) in production part.
```
if ! yarn lint-all; then
    echo "Lint check failed"
    exit ${BUILD_FAILURE}
fi
```

Reply:
agree, believe i've already done that, but through a CCI step?  like so?

https://github.com/okta/okta-developer-docs/blob/4373ecf9668429dc76198ad20df9882882a8c8da/.circleci/config.yml#L50-L55

Reply:
Okay let's leave it.
It's either we run the linting two times then builds on CCI and Bacon will have all the checks but we're wasting resources. Or we run everything on CCI and for Bacon we know that all checks are green already.

Reply:
Sorry @emikhasyak-okta I was missing context, but re-reading I think I'm groking. You're saying because it's a package setting, we're actually running that build step in Bacon ALSO, and so we were properly linting in Bacon. And since i separated it out here in CCI, it's now NOT linting in Bacot - which could be good (no duplicate resourcing) or bad (not good coverage?)

Alternatively, could do this same lint separation to a new step in Bacon if we think useful?

otherwise if this is just confusing happy to revert! hoping i'm understanding the root query here.

Reply:
Yes, we're running the same build command in bacon from a shell script. Let's keep your change as it is, since we already linting everything in CCI.